scripts/run_docker.sh

```diff
@@ -5,14 +5,13 @@ export NEAR_HOME=/srv/near
 
 NEARD_FLAGS=${NEAR_HOME:+--home=$NEAR_HOME}
 
-if [[ -z {INIT} ]]
+if [[ -n "${INIT}" ]]
```

Comment:
I’d actually consider changing this (and the other `if` below) to `if [ -v "$INIT" ]` so that we can change shebang from bash to `#!/bin/sh`.

Reply:
you mean `if [ -n "$INIT" ]`, right? 
Because iirc `-v` is an option for new version of bash only.